input/standardized-verification/bo/lwr_cool

```diff
@@ -0,0 +1,102 @@
+1991
```

Comment:
What's the meaning of this file, and why does one need these + signs at the beginning of each line?